README.md

```diff
@@ -28,9 +28,9 @@ Today, there are many derived Plasmas, like
 - Plasma-Chamber: Cryptoeconomics Lab's opensource project inspired by Prime. 
 - Plasma-Snapps: implemented ZK-S[T|N]ARKs
 
-Plasm has Plasma-Abstract data structures by which the user can custormize plagable Plasma solutions. In addtion to that, it has the Rust implementation for Plasma solutions. 
+Plasm has Plasma-abstract data structures by which the user can custormize plagable Plasma solutions. In addtion to that, it has the Rust implementations of Plasma solutions. 
```

Comment:
Plasm provides a Plasma-abstract data structure which is a combination of Plasma solutions. Also Plasm provides a Rust implementations of Plasma solutions.